scripts/Jenkinsfiles/python

```diff
@@ -137,17 +140,17 @@ pipeline {
                         ci_branch = "${ghprbSourceBranch}"
                         target_branch = "origin/${ghprbTargetBranch}"
                     } else {
-                        git_branch = 'master'
-                        git_refspec = "+refs/heads/master:refs/remotes/origin/master"
-                        ci_branch = "master"
-                        target_branch =  "origin/master"
+                        git_branch = "${BRANCH_NAME}"
+                        git_refspec = "+refs/heads/${BRANCH_NAME}:refs/remotes/origin/${BRANCH_NAME}"
+                        ci_branch = "${BRANCH_NAME}"
+                        target_branch =  "${BRANCH_NAME}"
```

Comment:
It looks like "origin/" was dropped here; was that intentional?

Reply:
Unintentional. It doesn't _realllly_ matter since I edited the script to be able to handle a branch with or without origin, but I'll add it in.